vizro-core/tests/unit/vizro/models/test_base.py

```diff
@@ -19,6 +19,11 @@ class ChildZ(vm.VizroBaseModel):
     type: Literal["child_Z"] = "child_Z"
 
 
+class ChildWithForwardRef(vm.VizroBaseModel):
+    type: Literal["child_forward_ref"] = "child_forward_ref"
```

Comment:
```suggestion
    type: Literal["child_with_forward_ref"] = "child_with_forward_ref"
```

Reply:
(would need find and replace throughout the file since it's in some other places too)